### PR TITLE
Daniel lab04 blackjack advice

### PR DESCRIPTION
**Pull request by DanielD6464**
Lab 4 Blackjack Finished

### CODE REVIEW

File: code/daniel/04_BlackJackAdvice/daniel-Lab4-Blackjack Advice.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+# What's your first card? Q
+# What's your second card? J
+# What's your third card? A
+# 21 Blackjack!
+#=============================================================
+
+import random
+
+Facecards = {
```
Comment (brucestull) [inline code/daniel/04_BlackJackAdvice/daniel-Lab4-Blackjack Advice.py:33]: * For Python, variables like `Facecards` should be in snake_case (`face_cards`). See [PEP8 Naming Conventions](https://peps.python.org/pep-0008/#naming-conventions)

File: code/daniel/04_BlackJackAdvice/daniel-Lab4-Blackjack Advice.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+    "7": 7,
+    "6": 6,
+    "5": 5,
+    "4": 4,
+    "3": 3,
+    "2": 2,
+    "A": 1
+}
+
+Question1 = input("What's your first card?: ")
```
Comment (brucestull) [inline code/daniel/04_BlackJackAdvice/daniel-Lab4-Blackjack Advice.py:49]: * Better naming would be `question_1` etc. (`snake_case`).

File: code/daniel/04_BlackJackAdvice/daniel-Lab4-Blackjack Advice.py [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+Question2 = input("What's your second card?: ")
+Question3 = input("What's your third card?: ")
+cardTotal = 0
```
Comment (brucestull) [inline code/daniel/04_BlackJackAdvice/daniel-Lab4-Blackjack Advice.py:52]: * `cardTotal` is camelCase (usually in Javascript). Use `snake_case` in Python. So `cardTotal` should be `card_total`.

**Review by brucestull — CHANGES_REQUESTED** [review]
[Blackjack Advice](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/04%20Black%20Jack%20Advice.md)

**Needs Work**

Comments:
* Need to change variables to `snake_case`.

**Review by nathanBonilla — DISMISSED** [review]
Hey Daniel, great job with your lab, I loved your utilization of dictionaries!
What I'd like to see before I approve your PR is either input validation, or final error.
What I mean is that, when I enter an invalid character, the program will still continue and output a result as if the invalid character didn't exist:
 - I'll enter 10, !, 10
 - The output will be a 20 even though there's a invalid card in my hand
There's two ways to tackle this.  Either ensure that I don't enter an invalid card, or let me know that my answer doesn't matter because there's an invalid card in my hand.

Once you make that fix, you're golden!

**Comment by DanielD6464** [comment]
Bruce said to grade lab 4 and ignore lab 7.

**Comment by brucestull** [comment]
Great job, Daniel!

[Blackjack Advice](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/04%20Black%20Jack%20Advice.md)

**Completed**

**Review by brucestull — APPROVED** [review]

**Pull request merged**